src/provisioner/log.py

```diff
@@ -25,23 +27,47 @@ class CortxProvisionerLog(Log):
     logger = None
 
     @staticmethod
-    def init(service_name, log_path=None, level='INFO', console_output=False):
+    def initialize(service_name, log_path=const.DEFAULT_LOG_PATH, level='INFO', console_output=True):
```

Comment:
Lets use some constant instead of 'INFO'
`DEFAULT_LOG_LEVEL="INFO"`

Reply:
Done.